### ISSUE TITLE
Imperfect work in server-paper1.19.1

### ISSUE THREAD

**Issue by Gu-f** [herby2212/FakePlayers#12]
When I execute the create command, an error is reported.
```
[00:03:33 INFO]: HelloMC issued server command: /fakeplayers create HelloMCS
[00:03:34 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'fakeplayers' in plugin FakePlayers v2.6
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.19.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.19.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.dispatchCommand(CraftServer.java:909) ~[paper-1.19.1.jar:git-Paper-110]
        at org.bukkit.craftbukkit.v1_19_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:59) ~[paper-1.19.1.jar:git-Paper-110]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[paper-1.19.1.jar:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:306) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:290) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2295) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$20(ServerGamePacketListenerImpl.java:2249) ~[?:?]
        at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1359) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:185) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1336) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1329) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1307) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1195) ~[paper-1.19.1.jar:git-Paper-110]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305) ~[paper-1.19.1.jar:git-Paper-110]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.List.add(Object)" because "de.Herbystar.FakePlayers.PlayerListHandler.NMS_PlayerListHandler.list" is null
        at de.Herbystar.FakePlayers.PlayerListHandler.NMS_PlayerListHandler.addCustomOnlinePlayer(NMS_PlayerListHandler.java:161) ~[FakePlayers-2.6.jar:?]
        at de.Herbystar.FakePlayers.Commands.onCommand(Commands.java:25) ~[FakePlayers-2.6.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.19.1-R0.1-SNAPSHOT.jar:?]
        ... 23 more
```
But the command came into effect.   
The added players can be seen in the TAB, but they are cleared after the OP exits.  
It seems that the working is not good because of the console error.

**Comment by herby2212** [comment]
Greetings,
thanks for the report. I will take a look into it.

**Comment by herby2212** [comment]
Fixed with release [2.7](https://github.com/herby2212/FakePlayers/releases/tag/2.7).

**Comment by Stage4000** [comment]
I feel you may have prematurely closed these threads relating to 1.19.2

Verions: 2.7

[14:18:05] [Server thread/INFO]: Stage4000 issued server command: /fakeplayers create randae
[14:18:05] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'fakeplayers' in plugin FakePlayers v2.7
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.dispatchCommand(CraftServer.java:833) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at org.bukkit.craftbukkit.v1_19_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:50) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:298) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:282) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.network.PlayerConnection.b(PlayerConnection.java:1944) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.network.PlayerConnection.lambda$18(PlayerConnection.java:1909) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.util.thread.IAsyncTaskHandler.b(SourceFile:68) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:157) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1150) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:131) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.bh(MinecraftServer.java:1129) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1122) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:140) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1106) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1017) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:293) ~[spigot-1.19.2-R0.1-SNAPSHOT.jar:3575-Spigot-450dcaa-370eece]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "de.Herbystar.FakePlayers.PlayerListHandler.PlayerListHandler.addCustomOnlinePlayer(String)" because "de.Herbystar.FakePlayers.Main.instance.playerListHandler" is null
	at de.Herbystar.FakePlayers.Commands.onCommand(Commands.java:25) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	... 23 more
